entries-api/README.md

```diff
@@ -0,0 +1,8 @@
+Tests for the [Files and Directory Entries API](https://github.com/wicg/entries-api)
+
+* [Explainer](https://github.com/WICG/entries-api/blob/gh-pages/EXPLAINER.md)
+* [Draft Specification](https://wicg.github.io/entries-api/)
+
+Unfortunately, most of the tests are **manual** and require drag-and-drop of test data.
+They can be run via [w3c-test.org](http://w3c-test.org/entries-api/), but a local
+clone of the repo is required for access to the `entries-api/support` files.
```

Comment:
It wasn't obvious to me until looking at the entries-api/support directory why you would need access to those files. Maybe explicitly mention that the test data it is talking about lives in that directory?